wix/main.wxs

```diff
@@ -0,0 +1,222 @@
+                            DiskId='1'
+                            Source='target\release\launchscript.exe'
+                            KeyPath='yes'/>
+                    </Component>
+                    <Component Id='nodeBinary' Guid='*' Win64='$(var.Win64)'>
+                        <File
+                            Id='nodeEXE'
+                            Name='node.exe'
+                            DiskId='1'
+                            Source='target\release\node.exe'
+                            KeyPath='yes'/>
+                    </Component>
+                    <Component Id='npmBinary' Guid='*' Win64='$(var.Win64)'>
+                        <File
+                            Id='npmEXE'
+                            Name='npm.exe'
+                            DiskId='1'
+                            Source='target\release\npm.exe'
+                            KeyPath='yes'/>
+                    </Component>
+                    <Component Id='npxBinary' Guid='*' Win64='$(var.Win64)'>
+                        <File
+                            Id='npxEXE'
+                            Name='npx.exe'
+                            DiskId='1'
+                            Source='target\release\npx.exe'
+                            KeyPath='yes'/>
+                    </Component>
+                    <Component Id='yarnBinary' Guid='*' Win64='$(var.Win64)'>
+                        <File
+                            Id='yarnEXE'
+                            Name='yarn.exe'
+                            DiskId='1'
+                            Source='target\release\yarn.exe'
+                            KeyPath='yes'/>
+                    </Component>
+                </Directory>
+            </Directory>
+            <Component Id='UserBinDir' Guid='96B83015-59EC-42E7-B85E-BDF6A1BC6198' Win64='$(var.Win64)'>
+                <RegistryValue
+                    Root='HKCU'
+                    Key='Software\[Manufacturer]\[ProductName]'
+                    Name='BinDir'
+                    Type='string'
+                    Value='[LocalAppDataFolder]Notion\bin'
+                    KeyPath='yes' />
+                <Environment
+                    Id='BINPATH'
+                    Name='PATH'
+                    Value='[LocalAppDataFolder]Notion\bin'
+                    Permanent='no'
+                    Part='first'
+                    Action='set'
+                    System='no' />
+            </Component>
+        </Directory>
+
+        <Feature Id='MainProgram'>
+            <ComponentRef Id='InstallDir'/>
+            <ComponentRef Id='UserBinDir'/>
+            <ComponentRef Id='notionBinary'/>
+            <ComponentRef Id='launchbinBinary'/>
+            <ComponentRef Id='launchscriptBinary'/>
+            <ComponentRef Id='nodeBinary'/>
+            <ComponentRef Id='npmBinary'/>
+            <ComponentRef Id='npxBinary'/>
+            <ComponentRef Id='yarnBinary'/>

```

Comment:
You could group these in place in a `ComponentGroup` and then only add one `ComponentGroupRef` here, which would be a little less verbose and thus easier to maintain if the list of files changes. No other technical benefit tho.

Reply:
Good idea, I didn't even know `ComponentGroupRef` existed!